__tests__/app.test.js

```diff
@@ -0,0 +1,42 @@
+const app = require("../app/app.js");
+
+const request = require("supertest");
+const seed = require("../db/seeds/seed.js");
+const db = require("../db/connection.js");
+const {
+  topicData,
+  userData,
+  articleData,
+  commentData,
+} = require("../db/data/test-data/index.js");
+beforeEach(() => seed({ topicData, userData, articleData, commentData }));
+
+afterAll(() => {
+  db.end();
+});
+describe("GET /api/topics", () => {
+  test("API healthcheck ", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
+  test("getting all topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual([
```

Comment:
There may be a point where a topic gets removed/gets added to the database. We should be testing in a more dynamic way. Make an assertion for how many topics are coming back and then make another assertion for the properties we expect there to be in each topic object in the database